core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -4004,111 +4004,135 @@ definitions:
         items:
           type: string
 
-  AttributeUpdateOperation:
+  AttributeUpdateOperationArray:
     description: ''
+    type: array
+    items:
+      $ref: '#/definitions/AtttributeUpdateOperation'
+    example: >
+      [
+        {
+          "op": "AddUpdateAttribute",
+          "attributeName": "string",
+          "addUpdateAttribute": "string"
+        },
+        {
+          "op": "RemoveAttribute",
+          "attributeName": "string"
+        },
+        {
+          "op": "AddListMember",
+          "attributeListName": "string",
+          "newMember": "string"
+        },
+        {
+          "op": "RemoveListMember",
+          "attributeListName": "string",
+          "removeMember": "string"
+        },
+        {
+          "op": "CreateAttributeEntityReferenceList",
+          "attributeListName": "string"
+        },
+        {
+          "op": "CreateAttributeValueList",
+          "attributeName": "string"
+        }
+      ]
+
+  AtttributeUpdateOperation:
+    discriminator: op
+    required:
+      - op
     properties:
-      Example payload for AddUpdateAttribute:
-        $ref: '#/definitions/AddUpdateAttribute'
-      Example payload for RemoveAttribute:
-        $ref: '#/definitions/RemoveAttribute'
-      Example payload for AddListMember:
-        $ref: '#/definitions/AddListMember'
-      Example payload for RemoveListMember:
-        $ref: '#/definitions/RemoveListMember'
-      Example payload for CreateAttributeEntityReferenceList:
-        $ref: '#/definitions/CreateAttributeEntityReferenceList'
-      Example payload for CreateAttributeValueList:
-        $ref: '#/definitions/CreateAttributeValueList'
+        op:
+          type: string
+          description: The operation to perform on the attribute
+          enum:
+            - AddUpdateAttribute
+            - RemoveAttribute
+            - AddListMember
+            - RemoveListMember
+            - CreateAttributeEntityReferenceList
+            - CreateAttributeValueList
 
   AddUpdateAttribute:
-    description: ''
+    allOf:
+      - $ref: '#/definitions/AtttributeUpdateOperation'
+      - properties:
+          attributeName:
+            type: string
+            description: The name of the attribute
+          addUpdateAttribute:
+            type: string
```

Comment:
nit: attributes can be any of the AttributeValue types, in json this is string, number, or bool